### ISSUE TITLE
Fix the Suggestions listbox labeling

### ISSUE THREAD

**Issue by afercia** [pacocoursey/cmdk#196]
It appears the Listbox with suggestions is incorrectly labeled;

https://github.com/pacocoursey/cmdk/blob/05199cbad94a800ee98510645f138537f368dc85/cmdk/src/index.tsx#L795-L803

- `aria-label="Suggestions"` is a hardcoded English text. This is not what most consumers need, as cmdk is used in multi-lingual applications that get translated in languages other than English. I guess this should be a prop with maybe a fallback to "Suggestions" if the prop is null.
- `aria-labelledby={context.inputId}` overrides `aria-label` anyways. The usage of both `aria-label` and `aria-labelledby` seems a bit pointless.
- However, `aria-labelledby={context.inputId}` points to the input id. As such:
  - The `aria-labelledby` points to an element that is initially 'empty' i.e. the input has no value. At this point, the suggestions Listbox is unlabelled.
  - As soon as users type something in the input field, the actual labelling of the suggestions Listbox is the value entered by users. This doesn't seem to be an appropriate labeling.

Proposed remediation:
- It would be way simpler to entirely remove `aria-labelledby` and only use `aria-label`, with a prop and fallback value.

To reproeuce:
- Go to https://cmdk.paco.me/
- Inspect the element with `role="listbox"` in your browser's dev tools > Accessibility tab
- Observe the accessible name of the element is an empty string (the element is unlabelled)
- Observe aria-labelledby overrides arira-label
- Type something in the input field
- Observe the element accessible bame is now the value entered in the field.

Screenshot from the current demo at https://cmdk.paco.me/

Initially unlabelled:

![Screenshot 2023-12-01 at 12 30 30](https://github.com/pacocoursey/cmdk/assets/1682452/e206d973-cad4-4b3a-8b13-65b5d4ef5cc0)

Wrongly labelled with the user input value:

![Screenshot 2023-12-01 at 12 16 05](https://github.com/pacocoursey/cmdk/assets/1682452/ce13a4d2-68ec-45e3-aff3-143077c45c5c)

For what is worth, I contribute to WordPress development as a core committer specialized in web accessibility. wordPress does use cmdk in the Gutenberg editor and we'd love to have cmdk as accessible as possible. ❤️
